content/docs/command-reference/move.md

```diff
@@ -161,3 +165,57 @@ $ tree
     │   └── foo
     └── data3.dvc
 ```
+
+## Example: manually renaming stage outputs
+
+`dvc move` doesn't support renaming of <abbr>outputs</abbr> in a stage, you will
+have to do that manually. Let's look at our sample workspace and `dvc.yaml`:
+
+```dvc
+$ tree
+.
+├── dvc.lock
+├── dvc.yaml
+├── keras.h5
+├── train.py
+└── ...
+
+$ cat dvc.yaml
+stages:
+  train:
+    cmd: python train.py
+    deps:
+    - data.csv
+    - train.py
+    params:
+    - epochs
+    - lr
+    outs:
+    - keras.h5
+```
+
+In this example we have to rename `keras.h5` as `model.h5`. First change the
+name of model file in code (`train.py`). Next change output name in `train`
+stage of `dvc.yaml`. After this change `/keras.h5` to `/model.h5` in
+`.gitignore` file. Lastly, rename the existing model file:
+
+```dvc
+$ mv keras.h5 model.h5
+```
+
+In final step run `dvc commit` to save the changes and update the `dvc.lock`
+file:
+
+```dvc
+$ dvc commit -f
+```
+
+Alternatively, you can also use `dvc run -f` to overwrite the changes in the
+stage in `dvc.yaml`:
```

Comment:
the stage changes (simple/direct language, and avoids using "in" twice in a row).